dvc/command/get.py

```diff
@@ -12,12 +12,13 @@
 class CmdGet(CmdBaseNoRepo):
     def _show_url(self):
         from dvc.api import get_url
+        from dvc.ui import ui
 
         try:
             url = get_url(
                 self.args.path, repo=self.args.url, rev=self.args.rev
             )
-            logger.info(url)
+            ui.write(url, force=True)
```

Comment:
Changed `get --show-url` to always print the url, even in `-q` mode.

Reply:
For the record: this goes per https://clig.dev/